python/data_extraction/export_data.py

```diff
@@ -105,7 +109,9 @@ def __bool__(self):
         def __iter__(self):
             return self.iterator
 
-    obj = { "items": StreamContent(iterator) }
+    obj = {"_meta": {"date": str(datetime.datetime.today()),
+                     "code": Repo(search_parent_directories=True).head.commit.hexsha},
+           "items": StreamContent(iterator)}
```

Comment:
My preference would be to indent this like:
```
    obj = {
        "_meta": {
            "date": str(datetime.datetime.today()),
            "code": Repo(search_parent_directories=True).head.commit.hexsha
        },
        "items": StreamContent(iterator)
}
```

Mostly because this is how I'm used to doing it, but also because the lines will be shorter.

Reply:
I'm happy to do either way. Should we add a linter to help with this sort of thing?

Reply:
Yes, adding some kind of linting would be good (but probably in a new Pull Request).